docs/dev-tools/console/console.asciidoc

```diff
@@ -155,74 +152,106 @@ GET /locations/_search
+Auto-indent current request::
+`Ctrl/Cmd` + `I`
+
+Jump to next request end::
+`Ctrl/Cmd` + `↓`
+
+Jump to previous request end::
+`Ctrl/Cmd` + `↑`
+
+Open documentation for current request::
+`Ctrl/Cmd` + `/`
+
+Run current request::
+`Ctrl/Cmd` + `Enter`
+
+Apply current or topmost term in autocomplete menu::
+`Enter` or `Tab`
+
+Close autocomplete menu::
+`Esc`
+
+Navigate items in autocomplete menu::
+`↓` + `↑`
 
-[role="screenshot"]
-image::dev-tools/console/images/formatted-request.png["Formatted request", width=75%]
 
-If you select *Auto indent* on a request that is already well formatted,
-*Console* collapses the request body to a single line per document.
-This is helpful when working with the {es} {ref}/docs-bulk.html[bulk APIs].
+
+[float]
+[[console-view-api]]
+==== View API docs
+
+To view the documentation for an API endpoint, select the request, then open the contextual menu and select
+*Open API reference*.
 
 [float]
 [[console-request]]
-=== Submit requests
+=== Run requests
+
+When you're ready to run a request, select the request, and click the play button.
 
-When you're ready to submit the request to {es}, click the green triangle.
+The result of the request execution is displayed in the response panel, where you can see:
 
-You can select multiple requests and submit them together.
-*Console* sends the requests to {es} one by one and shows the output
-in the response pane. Submitting multiple requests is helpful
+* the JSON response
+* the HTTP status code corresponding to the request
+* The execution time, in ms.
+
+TIP: You can select multiple requests and submit them together.
+*Console* executes the requests one by one. Submitting multiple requests is helpful
 when you're debugging an issue or trying query
 combinations in multiple scenarios.
 
+
+
 [float]
-[[console-view-api]]
-=== View API docs
+[[import-export-console-requests]]
+=== Import and export requests
+
+You can export requests:
+
+* **to a TXT file**, by using the **Export requests** button. When using this method, all content of the input panel is copied, including comments, requests, and payloads. All of the formatting is preserved and allows you to re-import the file later, or to a different environment, using the **Import requests** button. 
++
+TIP: When importing a TXT file containing Console requests, the current content of the input panel is replaced. Export it first if you don't want to lose it, or find it in the **History** tab if you already ran the requests.
 
-To view the documentation for an API endpoint, click
-the action icon (image:dev-tools/console/images/wrench.png[]) and select
-*Open documentation*.
+* by copying them individually as **curl**, **JavaScript**, or **Python**. To do this, select a request, then open the contextual menu and select **Copy as**. When using this action, requests are copied individually to your clipboard. You can save your favorite language to make the copy action faster the next time you use it.
```

Comment:
nit: I think it might be clearer to separate "import/export" and "copying individual requests in diff languages" into two different subheadings

Reply:
I considered it but this page already has too many too small sections and I find it annoying to read. I'll leave this as is for now